Source/doomstat.h

```diff
@@ -286,17 +286,22 @@ extern  int             bodyqueslot;
 
 extern int    skyflatnum;
 
+/*
```

Comment:
I guess you can just outright remove these lines.

Reply:
All commented parts are intended to be removed, this is a work in progress.